bpython/test/test_line_properties.py

```diff
@@ -7,7 +7,8 @@
     current_method_definition_name, current_single_word, \
     current_string_literal_attr, current_indexed_member_access_identifier, \
     current_indexed_member_access_identifier_with_index, \
```

Comment:
current_indexed_member_access_identifier, current_indexed_member_access_identifier_with_index, and current_indexed_member_access_member are all no longer used and should be deleted.